fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(users => {
+        this.setState({ isLoading: false });
```

Comment:
You need to update the `users` in state and `isLoading`. Make sure you are testing your code when you make updates.
```JAVASCRIPT
this.setState({ users: users, isLoading: false });
```